external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitter.java

```diff
@@ -279,6 +293,25 @@ private long seek(TopicPartition tp, KafkaTridentSpoutBatchMetadata lastBatchMet
         return fetchOffset;
     }
 
+    /**
+     * Seek the consumer to offset corresponding to startTimeStamp.
+     */
+    private void seekOffsetByStartTimeStamp(TopicPartition tp) {
+        long startTimeStampOffset;
+        try {
+            startTimeStampOffset =
+                    consumer.offsetsForTimes(Collections.singletonMap(tp, startTimeStamp)).get(tp).offset();
+        } catch (IllegalArgumentException e) {
```

Comment:
Why do we need to catch and handle these exceptions? Can't we just not catch them and it'll work?

Reply:
I was thinking , it would come up better in the UI, with ReportedFailedExceptions. But I can take it off, if it does not add value.

Reply:
Actually at this point, I guess, just letting the exceptions go, without catching should be fine. Removing the catch statements